MergeSharpWebAPI/Hubs/FrontEndHub.cs

```diff
@@ -12,13 +12,13 @@ public class FrontEndHub : Hub<IFrontEndClient>
 {
     public override Task OnConnectedAsync()
     {
-        _ = UserHandler.ConnectedIds.Add(Context.ConnectionId);
+        // _ = UserHandler.ConnectedIds.Add(Context.ConnectionId);
         return base.OnConnectedAsync();
     }
 
     public override Task OnDisconnectedAsync(Exception? exception)
     {
-        _ = UserHandler.ConnectedIds.Remove(Context.ConnectionId);
+        // _ = UserHandler.ConnectedIds.Remove(Context.ConnectionId);
         return base.OnDisconnectedAsync(exception);
```

Comment:
yoo what do these changes do? I'm assuming we overwrote the base functions but then we decided to not do that anymore and now we use the base functions?

Reply:
yea  you are right, we just use the base functions now because we dont need to overwrite them anymore. these functions can just be deleted i think